tools/osbuild-image-info

```diff
@@ -2511,7 +2515,7 @@ def discover_lvm(dev: str, parent: devices.Device, devmgr: devices.DeviceManager
         raise RuntimeError(res.stderr.strip())
 
     data = res.stdout.strip()
-    parsed = list(map(lambda l: l.split(";"), data.split("\n")))
+    parsed = list(map(lambda line: line.split(";"), data.split("\n")))
```

Comment:
(nitpick) I am personally not a fan of map/lambda, it seems ` [line.split(";") for line in data.split("\n")]` would also work and feels slightly more pythonic (to me) nowdays. But afaict we don't have tests so just blindly applying my suggestion would be dangerous. Lets just keep it for now.

Reply:
Didn't make this change since it works and is mostly a style choice.  Like @thozza said, let's clean it up when we have tests.